safe-global/safe-wallet-web
# feat: show Safe{Pass} points in header

## What this PR changes
- Shows the Safe{Pass} points in the header on Sepolia and Mainnet

## How to test it
- Open a Safe with points on Mainnet
- Check the header

## Screenshots
![Screenshot 2024-08-13 at 12 29 32](https://github.com/user-attachments/assets/3b140d77-128a-4b25-b144-7f0878a12292)
![Screenshot 2024-08-13 at 12 29 28](https://github.com/user-attachments/assets/4a9e7201-430c-4f97-9890-5abcaab2a539)


## Checklist
